python_modules/dagit/dagit/cli.py

```diff
@@ -186,7 +187,8 @@ def uploading_logging_thread():
 def start_server(instance, host, port, path_prefix, app, port_lookup, port_lookup_attempts=0):
     server = pywsgi.WSGIServer((host, port), app, handler_class=WebSocketHandler)
 
-    logger = default_system_logger("dagit")
+    configure_loggers()
+    logger = logging.getLogger("dagster.dagit")
```

Comment:
does this change the output of the logs to start with dagster.dagit instead of dagit? Is that what we want? seems like in many places this is supposed to be a module, but the top-level module is dagit not dagster.dagit right?

Reply:
refactored to "dagit"